presto-main/src/main/java/com/facebook/presto/execution/NodeResourceStatusConfig.java

```diff
@@ -35,4 +37,32 @@ public NodeResourceStatusConfig setRequiredWorkersActive(int requiredWorkersActi
         this.requiredWorkersActive = requiredWorkersActive;
         return this;
     }
+
+    @Min(0)
```

Comment:
Should these default to 1, and have a `@Min(1)` instead of 0?

Reply:
updated to 1